include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -624,9 +624,16 @@ namespace SuperiorMySqlpp { namespace LowLevel
              *
              * @return MYSQL_ROW with next row results. NULL if there are no more rows or on error.
              */
-            auto fetchRow() noexcept
+            auto fetchRow()
             {
-                return mysql_fetch_row(resultPtr);
+                auto *row = mysql_fetch_row(resultPtr);
+
+                if (row == nullptr && mysql_errno(resultPtr->handle) != 0)
```

Comment:
Consider using different fetchRow for store vs use fetch. 
Do we really need this condition for store mode?
Is store mode configurable by template? If so, then in store mode this method (and subsequently its caller) can be marked as noexcept.

Reply:
As far as I know, it is not configurable by template. However this should be trivial to implement.